examples/src/examples/graphics/reflection-planar/controls.mjs

```diff
@@ -0,0 +1,7 @@
+/**
+ * @param {import('../../../app/components/Example.mjs').ControlOptions} options - The options.
+ * @returns {JSX.Element} The returned JSX Element.
+ */
+export function controls({ fragment }) {
+    return fragment();
```

Comment:
For examples that don't specify controls, would it be better to allow for `controls.mjs` to be left unspecified?

Reply:
It just included it just in the case where someone would want to experiment with the controls since the code editor window doesn't show if there are no controls specified

Reply:
I have removed the empty files in favor of a template now 👍